src/controllers/manage/list/appointmentsListViewController.js

```diff
@@ -138,7 +138,7 @@ angular.module('bahmni.appointments')
             };
 
             $scope.translateAppointmentStatus = function (appointmentStatus) {
-                if(appointmentStatus === APPOINTMENT_STATUS_WAITLIST.status)
+                if(appointmentStatus === "WaitList")
```

Comment:
@Phanindra-tw why APPOINTMENT_STATUS_WAITLIST changed to "WaitList" ?

Reply:
@binduak APPOINTMENT_STATUS_WAITLIST is the object which was sent as part of Request body to backend. It had the status field. Now we decided that to backend wont send that param in Request body we wont have status inside APPOINTMENT_STATUS_WAITLIST object.